### DISCUSSION TITLE
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in .../node_modules/@prefresh/vite/package.json imported from .../node_modules/@preact/preset-vite/dist/esm/index.mjs

### DISCUSSION THREAD

**Discussion by abcd-arl** [preactjs/preact#4183] [Q&A] [answered]
Is anyone also experiencing this error? It happened out of nowhere without changing anything from my files (I could run my code 3 days ago.)

**Comment by JoviDeCroock** [comment, marked as answer]
This is for the prefresh repo and I'll fix it now :D

**Reply by abcd-arl** [reply]
I see. Thank you!
